string gear_state
uint64 timestamp 
uint64 source_time
